wbmalert.go

```diff
@@ -66,26 +67,30 @@ func createInitialSnapshot(website *website, wg *sync.WaitGroup) {
 	content, err := getWebsiteAsString(website)
 	if err == nil {
 		website.Snapshot = content
-		log.Println("Created initial snapshot for " + website.Name)
+		log.Println("[INFO]", "Created initial snapshot for "+website.Name)
```

Comment:
Structured logging is a great idea. Can you use the `log/slog` package instead of manually adding log levels?

Reply:
Hello Felix,
Sure, I have added slog as logger.